manifests/script-lab-react-prod.outlook.xml

```diff
@@ -57,7 +57,7 @@
             </DesktopSettings>
         </Form>
     </FormSettings>
-    <Permissions>ReadWriteMailbox</Permissions>
```

Comment:
Does this particular change adversely impact other scenarios?

Reply:
Not to my knowledge. We can take some time to verify, but it passed the store's validation, so I believe it works as expected

Reply:
And did you validate the append on send scenario. I think I tried this out a long time ago and it worked with the manifest change but can't remember for sure

Reply:
Yes we validated the Append on Send scenario. For some reason it wasn't working until we changed this permission.